x-pack/metricbeat/module/aws/rds/data.go

```diff
@@ -75,5 +76,34 @@ var (
 		"db_instance.class":                 c.Str("DatabaseClass"),
 		"db_instance.role":                  c.Str("Role"),
 		"db_instance.engine_name":           c.Str("EngineName"),
+
+		"aurora_bin_log_replica_lag": c.Float("AuroraBinlogReplicaLag"),
+
+		"aurora_global_db.replicated_write_io.bytes": c.Float("AuroraGlobalDBReplicatedWriteIO"),
+		"aurora_global_db.data_transfer.bytes":       c.Float("AuroraGlobalDBDataTransferBytes"),
+		"aurora_global_db.replication_lag.ms":        c.Float("AuroraGlobalDBReplicationLag"),
+
+		"aurora_replica.lag.ms":     c.Float("AuroraReplicaLag"),
+		"aurora_replica.lag_max.ms": c.Float("AuroraReplicaLagMaximum"),
+		"aurora_replica.lag_min.ms": c.Float("AuroraReplicaLagMinimum"),
+
+		"backtrack_change_records.creation_rate": c.Float("BacktrackChangeRecordsCreationRate"),
+		"backtrack_change_records.stored":        c.Float("BacktrackChangeRecordsStored"),
+
+		"backtrack_window.actual": c.Float("BacktrackWindowActual"),
+		"backtrack_window.alert":  c.Float("BacktrackWindowAlert"),
+
+		"storage_used.backup_retention_period.bytes": c.Float("BackupRetentionPeriodStorageUsed"),
+		"storage_used.snapshot.bytes":                c.Float("SnapshotStorageUsed"),
+
+		"cache_hit_ratio.buffer":     c.Float("BufferCacheHitRatio"),
+		"cache_hit_ratio.result_set": c.Float("ResultSetCacheHitRatio"),
+
+		"engine_uptime.sec":                        c.Float("EngineUptime"),
+		"volume_read_io_per_sec":                   c.Float("VolumeReadIOPs"),
```

Comment:
Maybe:

`volume.io_per_sec.read` and `volume.io_per_sec.write`?

Reply:
How about `volume.read.iops` and `volume.write.iops`? I think` iops` is the unit of these two metrics. That's why I put it at the end of the metric name.

Reply:
👍